whois-scheduler/src/test/java/net/ripe/db/whois/scheduler/task/export/ExportFileWriterFactoryTest.java

```diff
@@ -57,12 +55,11 @@ public void createExportFileWriters() {
 
         final File[] files = folder.toFile().listFiles();
         assertThat(files, not(nullValue()));
-        assertThat(files.length, is(3));
+        assertThat(files.length, is(2));
 
         for (final File file : files) {
             if (! (file.getAbsolutePath().endsWith("internal")
-                    || file.getAbsolutePath().endsWith("dbase")
-                    || file.getAbsolutePath().endsWith("dbase_new"))) {
+                    || file.getAbsolutePath().endsWith("dbase"))) {
                 fail("Unexpected folder: " + file.getAbsolutePath());
```

Comment:
can you refactor the code to assertThat that it is just dbase and internal in any order?

Reply:
done